installer/conf/omsagent.conf

```diff
@@ -76,6 +76,10 @@
 #  type filter_nagios_log
 #</filter>
 
+<filter collectd>
+  type filter_collectd
+</filter>
```

Comment:
This should go in it's own file so that it can be placed in the omsagent.d directory.
See changetracking.